### DISCUSSION TITLE
useAtomsDebugValue - list atoms in alpha order

### DISCUSSION THREAD

**Discussion by AjaxSolutions** [pmndrs/jotai#1567] [Q&A] [answered]
How can I list the atoms in alphabetical order?
I have a long list of atoms and having an alpha order would be helpful.

```
isDemoModeAtom
workspacesDataAtom
smasAtom
...
```

**Comment by Aslemammad** [comment, marked as answer]
We do not have an option for printing atoms in alphabetical order, and I'm not informed about React Devtools, so I do not know if it provides such an option for values. Still, we may also be able to implement this option in other devtools.

cc @dai-shi

**Reply by dai-shi** [reply]
Without deep thoughts, I think we can just sort on our end, if that helps.

**Reply by Aslemammad** [reply]
Yes, that'd be possible.

@AjaxSolutions Let us know if you are open to contribution, if not, we'll do it then.

**Comment by AjaxSolutions** [comment]
@Aslemammad Please go ahead and do it.

I have a couple hundred atoms and each atom has the debugLabel attached to it, so if you could sort this list on the debugLabel, that would be great.

The other somewhat less annoying issue is that this list includes intermediate atoms generated by selectAtom and loadable.  I'd prefer not to see these atoms, but there is probably not much you can do about it?

**Reply by dai-shi** [reply]
> there is probably not much you can do about it?

I think we could sort first with atoms with `debugLabel`. That may help to some extent.

**Reply by Aslemammad** [reply]
Agreed, then we defer those that do not contain debugLabel.

**Comment by AjaxSolutions** [comment]
Do you know if it is possible to modify atoms in React Dev Tools?

**Reply by dai-shi** [reply]
Being asked once, and as far as I remember, React Dev Tools doesn't have such a capability. It would be nice though.
Redux DevTools can do.
